pkg/workload/schemachange/query_util.go

```diff
@@ -18,9 +18,38 @@ import (
 	"github.com/jackc/pgx/v5"
 )
 
-var (
+const (
+	// descJSONQuery returns the JSONified version of all descriptors in the
+	// current database joined with system.namespace.
+	//
+	// id::int | schema_id::int | name::text | descriptor::json
+	descJSONQuery = `SELECT
+		descriptor.id,
+		"parentSchemaID" AS schema_id,
+		namespace.name AS name,
+		crdb_internal.pb_to_json('desc', descriptor) AS descriptor
+	FROM system.descriptor
+	JOIN system.namespace ON namespace.id = descriptor.id
+	WHERE "parentID" = (SELECT id FROM system.namespace WHERE name = current_database() AND "parentID" = 0)
```

Comment:
why do we have the `AND "parentID" = 0` here?

In other words, why isn't 
```
SELECT id FROM system.namespace WHERE name = current_database();
```
sufficient to find the ID of the current database?

Reply:
`"parentID" = 0` ensures that we're selecting a database form the namespace table. If we omitted that and someone created, say, a table `movr` in `cockroach demo`, the query you've posted will return multiple rows.